fiftyone/operators/store/models.py

```diff
@@ -0,0 +1,35 @@
+"""
+Store and key models for the execution store.
+"""
+
+from pydantic import BaseModel, Field
```

Comment:
Yeah we removed `pydantic` from `threed` before 3D release because we didn't want to add the dependency without a compelling enough reason. We were able to just use `dataclasses`.